draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -3683,6 +3681,8 @@ This section gives a straight-line implementation of the Shallue and van
 de Woestijne method for any Weierstrass curve of the form given in
 {{weierstrass}}.
 See {{svdw}} for information on the constants used in this mapping.
+Note that the sign of constant c3 below MUST equal 0. If sgn0(c3) = -1,
+c3 MUST instead be set to -c3.
```

Comment:
```suggestion

Note that the constant c3 below MUST be chosen such that sgn0(c3) = 0.
In other words, if the square-root computation returns a value cx such that
sgn0(cx) = 1, set c3 = -cx; otherwise, set c3 = cx.
```

Reply:
Is this slightly clearer?

Also, nowadays sgn0 returns only 0 or 1, so I think there was a typo here.

Reply:
it's clear

Reply:
Yeah, works for me!